docker-app/qfieldcloud/core/models.py

```diff
@@ -424,6 +424,28 @@ def avatar_url(self):
         else:
             return None
 
+    # TODO: definitely need to cache these, otherwise we must recompute from ALL projects each time !
+    def storage_quota_left(self) -> float:
```

Comment:
I think there's some kind of convention that says that property shouldn't be used for heavy lifting methods (as otherwise you can easily inadvertently make many queries), but will change once cache is in place